test/integration/action/action-integration.spec.js

```diff
@@ -228,6 +228,27 @@ describe('Action Integration Tests', function() {
       expect(store2.lndReady, 'to be true');
     });
 
+    it('should wait for autopilotReady', async () => {
+      await grpc1.initAutopilot();
+      expect(store1.autopilotReady, 'to be true');
+      await grpc2.initAutopilot();
+      expect(store2.autopilotReady, 'to be true');
+    });
+
+    it('should be able to toggle autopilot', async () => {
+      let status = await grpc1.sendAutopilotCommand('status');
+      expect(status.active, 'to be true');
+      await grpc1.sendAutopilotCommand('modifyStatus', { enable: false });
+      status = await grpc1.sendAutopilotCommand('status');
+      expect(status.active, 'to be false');
+
+      status = await grpc2.sendAutopilotCommand('status');
+      expect(status.active, 'to be true');
+      await grpc2.sendAutopilotCommand('modifyStatus', { enable: false });
+      status = await grpc2.sendAutopilotCommand('status');
+      expect(status.active, 'to be false');
+    });
+
```

Comment:
Nice that we have integration tests. But we should test the higher level `setting.toggleAutopilot()` action instead of the grpc and check that `store.settings.autopilot` is set correctly. We should also not forget to test turning autopilot on again.

Reply:
Are you saying remove [these](https://github.com/lightninglabs/lightning-app/pull/1000/files#diff-a2c88e3a85c1882dbd0b41495eb2b965R238) tests and add testing for autopilot-on to [these](https://github.com/lightninglabs/lightning-app/pull/1000/files#diff-226a70f557ce7896930bef43709bcd75R100) tests?

Reply:
fixed